docs/guides/chip_tool_guide.md

```diff
@@ -314,39 +324,41 @@ Having completed all previous steps, you have the Matter device successfully
 commissioned to the network. You can now test the device by interacting with
 Data Model clusters.
 
-For instance, in case of the lighting application, the application has the
-On/Off and Level Control clusters implemented. This means that you can test it
-by toggling the bulb (using the `onoff` cluster commands) or manipulating its
-brightness (using the `levelcontrol` cluster commands).
+#### Example: Matter Lighting Application Example
 
-Use the following command pattern to toggle the LED state:
+In case of the Matter Lighting Application Example we referenced in step 1, the
+application implements the On/Off and Level Control clusters. This means that
+you can test it by toggling the bulb (using the `onoff` cluster commands) or by
+manipulating its brightness (using the `levelcontrol` cluster commands):
 
-```
-$ ./chip-tool onoff toggle <node_id> <endpoint_id>
-```
+-   Use the following command pattern to toggle the LED state:
```

Comment:
Minor: we should not assume that onoff cluster state will be corresponding to some LED state on the board in all platforms. Maybe we could rephrase it in a following way?
```suggestion
-   Use the following command pattern to toggle the OnOff attribute state (e.g. visualized by the LED state):
```